src/hooks/useFlags/flagsmithProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import flagsmith from "flagsmith";
+import { FlagsmithProvider } from "flagsmith/react";
+import React from "react";
+
+export const Provider = ({ children }: { children: React.ReactElement }) => (
```

Comment:
**issue**: add this provider to storybook `Decorator`

I think you will need to add this provider to [Decorator](https://github.com/saleor/saleor-dashboard/blob/d602ac906b324a4d1e659557956bc997176752e8/src/storybook/Decorator.tsx#L1) as currently some stories depends on this to be in component tree.

Example: https://2887-feature-flags-abstraction.dashboard.saleor.rocks/storybook/index.html?path=/story/views-apps-app--default